itou/employee_record/models.py

```diff
@@ -106,7 +109,8 @@ def __str__(self):
     def save(self, *args, **kwargs):
         if self.pk:
             self.updated_at = timezone.now()
-
+        if self.job_seeker_profile:
+            self.job_seeker_profile.save()
```

Comment:
Il faut enregistrer systématiquement le profil ? Je ne me souviens plus pourquoi il y a une liaison si forte avec `EmployeeRecord` ?

Reply:
Ah ! Quel oeil de lynx. J'ai besoin de vos lumières sur le sujet.

C'est un souci technique que je ne m'explique pas et dont je voudrais discuter.

Une relation "One-to-One" ne peut être persistée directement que si l'objet "parent" a déjà été sauvegardé en base (possède une PK).

Reply:
Comme tu le dis dans une O2O le parent doit être persisté **avant** mais ici nous ne sommes pas dans ce cas car `job_seeker_profile` est n'est pas directement lié à `EmployeeRecord -> FK job_application -> FK job_seeker -> O2O jobseeker_profile`
donc c'est uniquement entre JobSeeker et JobSeekerProfile que nous avons cette problématique à la création.